src/app/vault/add-edit.component.html

```diff
@@ -132,9 +135,9 @@ <h2 class="modal-title" id="cipherAddEditTitle">{{title}}</h2>
                                 <label for="loginUri{{i}}">{{'uriPosition' | i18n : (i + 1)}}</label>
                                 <div class="input-group">
                                     <input class="form-control" id="loginUri{{i}}" type="text"
-                                        name="Login.Uris[{{i}}].Uri" [(ngModel)]="u.uri"
+                                        name="Login.Uris[{{i}}].Uri" [(ngModel)]="u.uri" [disabled]="cipher.isDeleted"
                                         placeholder="{{'ex' | i18n}} https://google.com" appInputVerbatim>
-                                    <div class="input-group-append">
+                                    <div class="input-group-append" *ngIf="!cipher.isDeleted">
```

Comment:
I don't see any harm in leaving these available still.

Reply:
The general idea that we discussed was to make everything "read-only" for items in the trash, and leaving edit fields, add/remove buttons, etc. available on screen would give the impression that you can still do things with the cipher and have them persist (but they would not actually persist/save/update).

Reply:
Fixed